test/compose/select-display-columns/enabled

```diff
@@ -1,2 +1,2 @@
 #!/bin/sh
-[ $(echo -n "😄😊😉😍" | wc -m) == 4 ] && locale | grep LC_CTYPE | grep -qi 'utf-*8'
+[ $(echo -n "😄😊😉😍" | wc -m) = '4' ] && locale | grep LC_CTYPE | grep -qi 'utf-*8'
```

Comment:
This baffles me. According to everything I know about shell-script parsing, this change should make no difference — `[` is an ordinary command, `4` is an ordinary word, and quoting words doesn't change their meaning unless they include metacharacters like `$` or `\` and `4` isn't one. But apparently in OpenBSD's shell this is a thing?

Reply:
This is a not a OpenBSD thing but a Linux thing or at least a Ubuntu thing. 

On Ubuntu 20.04 which uses the dash shell as the `sh` shell, I get this: 

```
$ ./enabled
./enabled: 2: [: 4: unexpected operator
$
```

Reply:
Interestingly, this is OK in OpenBSD!

In OpenBSD 7.0
```sh
$ [ 4 == '4' ]
$ echo $? 
0
```

But using `sh` (which is dash on Ubuntu) you get an error as mentioned in above comment. So the `enabled` script will be fine in OpenBSD. 

Fedora is fine because it uses bash as its `sh` and bash is quite forgiving in general.